style.css

```diff
@@ -0,0 +1,88 @@
+.container {
+  position: absolute;
+  background-image: url("img/mobile-icon.png");
+  background-color: #fff;
+  width: 375px;
+  margin: 0;
+  height: 30rem;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Please pay attention that the Minimum mobile screen size should be `375`px.

Reply:
Need clarification on this please